crates/rattler_conda_types/Cargo.toml

```diff
@@ -12,6 +12,7 @@ readme.workspace = true
 
 [features]
 default = ["rayon"]
+extras = []
```

Comment:
Lets rename this to `experimental_extras` to indicate that this is still experimental. We will remove the feature if we deem it stable enough.